pkg/work/spoke/options.go

```diff
@@ -4,19 +4,22 @@ import (
 	"time"
 
 	"github.com/spf13/pflag"
+	"open-cluster-management.io/api/cloudevents/generic/options/mqtt"
 )
 
 // WorkloadAgentOptions defines the flags for workload agent
 type WorkloadAgentOptions struct {
 	StatusSyncInterval                     time.Duration
 	AppliedManifestWorkEvictionGracePeriod time.Duration
+	MQTTOptions                            *mqtt.MQTTOptions
```

Comment:
Do we need to start webhook server if it is configured to run with mqtt option? I think we may not. https://github.com/open-cluster-management-io/ocm/blob/8a4c834ebf0f3c71ab9209572fb0b11cc942f5d5/cmd/work/main.go#L62

Reply:
this is options only for agent，the webhook-server is a sub command for work agent, we can control whether to start it outside

```
./work --help
Spoke Cluster Work

Available Commands:
  agent          Start the Work Agent
  completion     Generate the autocompletion script for the specified shell
  help           Help about any command
  manager        Start the Work Hub Manager
  webhook-server Start the work webhook server
```

to use this `work agent --mqtt-broker=...`